src/main/kotlin/com/exactpro/th2/check1/rule/RuleFactory.kt

```diff
@@ -45,23 +45,23 @@ import java.time.Instant
 import java.util.concurrent.ForkJoinPool
 
 class RuleFactory(
-    configuration: Check1Configuration,
-    private val streamObservable: Observable<StreamContainer>,
-    private val eventBatchRouter: MessageRouter<EventBatch>
+        configuration: Check1Configuration,
+        private val streamObservable: Observable<StreamContainer>,
+        private val eventBatchRouter: MessageRouter<EventBatch>
```

Comment:
According to the Kotlin code style the properties and parameters in the constructor should have a single indent (4 spaces instead of 8)